src/ScottPlot/Plot.cs

```diff
@@ -1156,7 +1156,7 @@ public void Clear(Predicate<Plottable> plottablesToClear)
                 double[] barYs = ys[i];
                 double[] barYerr = yErr?[i];
                 double[] barXs = DataGen.Consecutive(barYs.Length);
-                containsNegativeY = containsNegativeY ? containsNegativeY : barYs.Where(y => y < 0).Any();
+                containsNegativeY |= barYs.Where(y => y < 0).Any();
```

Comment:
I didn't know C# supported bitwise on booleans? Intriguing, I suspect somewhere buried deep in the MSIL there's a macro like this:

`#define true 1`
`#define false 0`

i.e. C# bools are not truly primitive types, but macros, as is the case in C, and is sort of true in C++.

Reply:
I've been doing a lot of bit operations lately (working on audio) and just learned about this one myself!

Don't forget `bool?` which can hold 3 possible states 😝 🙈